packages/chat-stateful-client/src/ChatContext.ts

```diff
@@ -300,6 +316,18 @@ export class ChatContext {
     }
   }
 
+  /* @conditional-compile-remove(teams-inline-images-and-file-sharing) */
+  private parseAttachments(threadId: string, message: ChatMessageWithStatus): void {
+    const attachments = message.content?.attachments;
+    if (message.type === 'html' && message.content?.message && attachments && attachments.length > 0) {
```

Comment:
minor: it's probably can be 1 `if` statement

Reply:
I'm a little against this since I worry it may become a little harder to read. I feel like this is getting too large as is.